src/process.rs

```diff
@@ -2,12 +2,14 @@ use anyhow::{anyhow, Result};
 use flate2::read::GzDecoder;
 use indicatif::{ProgressBar, ProgressStyle};
 use nohash::IntMap;
-use raad::le::*;
+use raad::le::{R, W};
+use serde::Deserialize;
 use std::{
     collections::{
         hash_map::Entry::{Occupied, Vacant},
         HashSet,
     },
+    hash::BuildHasherDefault,
```

Comment:
why? Default::default is clearer here

Reply:
i disagree-- BuildHasherDefault::default is quite meaningless, especially in the context of `with_hasher(Default::default())` < says build the hashmap with the default hasher.
`with_hasher(BuildHasherDefault::default()` says: build the hashmap with the build hashmap's default hasher-- its the same information, just more verbosely. the warning is okay, but its a false positive here.

Reply:
uhm fair ig uess, not much of difference anyway i'll revert in the next commit